src/components/timeline/Track.css

```diff
@@ -63,21 +63,29 @@
 }
 
 .timelineTrackCloseButton {
-  position: relative;
-  right: 5px;
   overflow: hidden;
-  width: 11px;
-  height: 11px;
-  padding: 0;
+  width: 15px;
+  height: 15px;
```

Comment:
This is a bit bigger compared to its previous icon now. I wonder if making it a bit smaller makes it a bit less crowded/flashing. But I don't mind either way tbh. :)

Reply:
True. It takes away the same amount of horizontal space from the label, but I'm happy to make it smaller. We can see how people feel about it.